traveler-swift-core/TravelerKit/Models/OrderResult.swift

```diff
@@ -64,12 +67,34 @@ public struct OrderResult: Decodable {
         }
 
         var orders = self.orders
+        var sequence = self.ordersSequence
 
         for (index, order) in result.orders {
             orders[index] = order
+            sequence[order] = index
         }
 
-        return OrderResult(total: result.total, orders: orders, fromDate: result.fromDate, toDate: result.toDate)
+        return OrderResult(total: result.total, orders: orders, ordersSequence: sequence, fromDate: result.fromDate, toDate: result.toDate)
+    }
+
+    /**
+     Updates `Order` in orders in self.
```

Comment:
Kinda confusing documentation

how about:

"Updates a specific `Order` in the internal storage"